.github/actions/hello-world/action.yml

```diff
@@ -0,0 +1,7 @@
+name: 'Hello World '
```

Comment:
I like the idea of having a lightweight action that we use as a quick canary to make sure things are working, but I don't like that it right now looks like placeholder code.

How about we  update this to explicitly be the thing we want it to be; call it something like a "startup status" action and have it print a message like "Hello! Your GitHub actions have started successfully". The description can read "A basic, lightweight action just to validate the actions code is good before we attempt the more expensive actions. Once caching is setup, we can remove this."

Reply:
sounds good!

Also to give context, since this technology is new to many, the idea is for this to be used as a tutorial like "Run your first action". I'll update the code with your comment!